webapp/src/extensionsBrowser.tsx

```diff
@@ -463,6 +469,11 @@ export const ExtensionsBrowser = (props: ExtensionsProps) => {
                                     role="button"
                                 />)}
                         </div>
+                        {searchComplete && extensionsToShow.length == 0 &&
+                            <div aria-label="Extension search results">
+                                <p>{lf("We couldn't find any extensions matching ")}{pxt.Util.rlf(`'${searchFor}'`)}</p>
```

Comment:
Use the format string (e.g. `{0}`)to append the string instead of putting it at the end. Different language grammars might require it to be placed elsewhere in the sentence.

Also, does it make more sense to put the rlf on the inner string? I mean, to not include the quotes when doing the lookup and just adding them to the result

Reply:
do we want to translation for the inner rlf at all? If I'm reading right that's a value from user input in the first place so it'd make or sense to leave as whatever they typed in (that is, if they type in 'controller' to the searchbox while in spanish it'd be weird to show it as 'Controles')

that said we could do an lf with `lf("'${0}'", searchFor)` to translate only the surrounding quotation marks to the given language's format

Reply:
Won't the inner string, `searchFor` already be in the appropriate language since it's user input?

Reply:
I've updated it to use format strings!